button.js

```diff
@@ -45,32 +42,62 @@ $closeButton.addEventListener('click', function () {
 
   hideModal();
 });
-var inputTask, inputMonth, inputDate, startTime, endTime;
+
+let tasks = [],
+  inputMonth = [],
+  inputDate = [],
+  startTime = [],
+  endTime = [];
```

Comment:
It seems that the functionality will be implemented in this way. good.

However, when deleting or modifying the todo later, because the elements are separated, the code is expected to be longer.
Also, there seems to be no performance benefit as the function is executed multiple times to load each element.
For example, if todo is saved 5 times, the function is called 25 times.